flathub.json

```
@@ -0,0 +1,3 @@
+{
+  "skip-arches": ["arm"]
```

Oh I should have mentioned: `aarch64` is basically 64-bit ARM
```suggestion
  "skip-arches": ["arm", "aarch64"]
```

---

Thanks! That should fix it now.